### DISCUSSION TITLE
`//| echo: fenced` option remains in rendered OJS code blocks

### DISCUSSION THREAD

**Discussion by allisonhorst** [quarto-dev/quarto-cli#2344] [Q&A]
Hi there! Thanks for everything you all do, Quarto is amazing!

I'm starting to work in some `{ojs}` cells in Quarto. When I use `#| echo: fenced` in R or Python blocks, it renders as expected, showing the fenced code delimiters but without showing the `#| echo: fenced` option within the rendered output. When I try `//| echo: fenced` in the `{ojs}` cell, the `//| echo: fenced` shows up in the rendered output. 

It works fine if `echo: fenced` is added in the YAML, so I'm only having this issue if it's added as an option in an individual {ojs} block. Not sure if I'm missing something or if this is unexpected. Thanks for any follow-up!

**Comment by ghost** [comment]
Thanks for asking this, I've moved it to issues: https://github.com/quarto-dev/quarto-cli/issues/2353
